### DISCUSSION TITLE
Question about using a Child Theme

### DISCUSSION THREAD

**Discussion by gesshoo** [gregsullivan/_tw#135] [Q&A]
Hello,

Quick question about the theme in production (live).

Would it be best practice to use a child theme in production so it can be easier to update the theme later? Do you recommend using a child theme with _tw?

**Comment by gregsullivan** [comment]
Hello—Thanks for posting this!

I don't recommend using a child theme. There's a documentation page on [keeping your theme updated](https://underscoretw.com/docs/updating/) that covers my philosophy there a little bit. Automattic has [discussed this approach as well](https://themeshaper.com/2013/10/11/dont-update-your-theme/), and I reference that post in the documentation page I linked above.

It can take a bit of a mindset change, but I see starter themes like _tw and _s as providing a base meant for direct editing, without child themes having a role in the process.

I'd be happy to answer any other questions you have about the above!

**Comment by gesshoo** [comment]
Thanks for the response. Glad you clarified that for me. 

I do have a question about the sidebar inside the footer. Is there a reason for this? Should I start a new discussion for the footer/sidebar?

**Reply by gregsullivan** [reply]
I'm comfortable answering your question in this thread!

There have been different approaches to sidebars in WordPress themes over the years. _s, for example, still calls `get_sidebar()` before `get_footer()` in templates like `single.php`.

_tw is based on _s, but with modernizations from Varia. Putting the sidebar in the footer mirror Varia's approach. If you don't need a sidebar, you can safely delete it.

Hope that helps!
